misc/pull-packcc.sh

```diff
@@ -6,4 +6,4 @@
 # License: GPL 2 or later
 
 cd $(git rev-parse --show-toplevel)
-git subtree pull --prefix misc/packcc https://github.com/universal-ctags/packcc.git master --squash
+git subtree pull --prefix misc/packcc https://github.com/arithy/packcc.git master --squash
```

Comment:
Are you sure you want to pull in complete packcc repository? Most of it (test/, build/) won't be really useful for anything here?

Unfortunately, I think it is not possible to pull only some files using `git subtree`. On the other hand, you're probably not using `git subtree` with `push` or `merge` and you `--squash` the commits, so it's not really necessary to use it at all. You could simply pull the desired files using `curl`. It could look like this:
```sh
cd $(git rev-parse --show-toplevel)
SHA="$(curl 'https://api.github.com/repos/arithy/packcc/branches/master' | jq -r .commit.sha)"
for FILE in src/packcc.c LICENSE README.md; do
    mkdir -p "$(dirname "misc/packcc/$FILE")"
    curl "https://raw.githubusercontent.com/arithy/packcc/$SHA/$FILE" -o "misc/packcc/$FILE"
done
git commit -m "Updated packcc to $SHA" -- misc/packcc
```

Reply:
I think getting the whole source tree is o.k.
We can avoid installing jq and curl for building ctags source tree if we use just git subtree.
We can expect the pull-packcc script works even if the directory layout of packcc is changed.
Of course, ctags' building script like Makefile.am must be revised when the layout is changed. However, we can reduce the items we have to think about when updating.

Reply:
You don't need curl and jq to build ctags, only if you want to update packcc. Or is the script called automatically somewhere in the build/release process? I thought you're running it manually only when you want to update.

But as I said, I don't really mind having more files here, you can leave this as is.